src/assets/styles/main.scss

```diff
@@ -82,3 +82,9 @@ html * {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
+
+button {
+  color: inherit;
+  border: none;
+  background-color: transparent;
```

Comment:
This needs `padding: 0;` and `cursor: pointer;`

Reply:
Done